### PR TITLE
Enable CD

### PR DESCRIPTION
**Pull request by batmat**
I'd like a closer-than-usual review of this one, given I've used the `revision` = `2.3.x` to reflect the fact this plugin is an API-plugin providing JAXB. (as per the part about _"For a component whose version number ought to reflect a release version of some wrapped component:"_ in the doc)

Cf. https://www.jenkins.io/doc/developer/publishing/releasing-cd/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     </scm>
     <properties>
         <revision>2.3.x</revision>
```
Comment (batmat) [inline pom.xml:30]: As explained, left on purpose.

Comment (jglick) [reply pom.xml:30]: https://github.com/jenkinsci/jaxb-plugin/blob/1cd47382d83631f6bef4a02a675622519cdcda05/pom.xml#L39-L53 does not smell right. Why are these versions different? Is there a BOM we could import? If not, and these three versions are all actually floating, I would recommend
```suggestion
        <revision>2.3</revision>
```

and then use Enforcer to verify that the versions of these three components all start with that prefix.

Comment (timja) [reply pom.xml:30]: they are different although looks like core does have a `2.3.0.1` available

Comment (jglick) [reply pom.xml:30]: Or is there one component (like `jaxb-impl`) which can be the sole import, where the others are transitive dependencies in suitably new versions?

Alternately, if there is not really a well-defined “version of JAXB”, then maybe the plain `${changelist}` is more appropriate.

Comment (basil) [reply pom.xml:30]: > Is there a BOM we could import?

=> #19

**Comment by batmat** [comment]
Whatever happens, https://github.com/jenkinsci/jaxb-plugin/pull/19 does look like the way to go as per the comments. So I'm closing this one here.

**Comment by basil** [comment]
But #19 is a prerequisite to this PR. It does not implement CD, nor does it intend to. This PR remains valid, but it needs to be rebased on top of #19 once #19 is merged.

**Pull request closed without merging**